obsRecorder.js

```diff
@@ -1,22 +1,27 @@
 const path = require('path');
 const { Subject } = require('rxjs');
 const { first } = require('rxjs/operators');
+const { ipcMain } = require('electron');
 
 const osn = require("obs-studio-node");
 
 let obsInitialized = false;
 
 // Init the library, launch OBS Studio instance, configure it, set up sources and scene
-function initialize() {
+function initialize(webContents) {
```

Comment:
It feels somewhat wrong for me, but I can't propose any better way to do it for now.

Reply:
I know, I tried to require it somehow, but the `remote` module is `undefined` when required in this context...